libsrc/leddevice/LedDeviceAdalightApa102.cpp

```diff
@@ -43,7 +35,7 @@ int LedDeviceAdalightApa102::write(const std::vector<ColorRgb> & ledValues)
 		_ledBuffer[5] = _ledBuffer[3] ^ _ledBuffer[4] ^ 0x55; // Checksum
 	}
 
-	for (unsigned iLed=1; iLed<=ledCount; iLed++) {
+	for (signed iLed=1; iLed<=_ledCount; iLed++) {
```

Comment:
Purely out of interest. Why the changed to a signed int? It would never become negative, would it?